geode-benchmarks/src/main/java/org/apache/geode/benchmark/topology/ClientServerTopology.java

```diff
@@ -69,18 +70,26 @@ public static void configure(TestConfig testConfig) {
     }
 
     String withSslArg = System.getProperty("withSsl");
-    if (withSslArg != null && withSslArg.equals("true")) {
-      logger.info("Configuring JVMs to run with SSL enabled");
-      testConfig.jvmArgs(CLIENT, Arrays.append(JVM_ARGS, WITH_SSL_ARGUMENT));
-      testConfig.jvmArgs(LOCATOR, Arrays.append(JVM_ARGS, WITH_SSL_ARGUMENT));
-      testConfig.jvmArgs(SERVER, Arrays.append(JVM_ARGS, WITH_SSL_ARGUMENT));
-    }
+    addToTestConfig(testConfig, withSslArg, WITH_SSL_ARGUMENT);
+
+    String withSecurityManagerArg = System.getProperty("withSecurityManager");
+    addToTestConfig(testConfig, withSecurityManagerArg, WITH_SECURITY_MANAGER_ARGUMENT);
```

Comment:
It turns out that `testConfig.jvmArgs` are additive so you don't need to merge them before calling this function.

Reply:
good catch